comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -11,6 +11,7 @@
       <packagereq type="default">rubygem-algebrick</packagereq>
       <packagereq type="default">rubygem-apipie-params</packagereq>
       <packagereq type="default">rubygem-chef-api</packagereq>
+      <packagereq type="default">rubygem-concurrent-ruby</packagereq>
```

Comment:
This version's already in the main Foreman repo (comps-foreman-rhel7), or did you mean to add the SCL variant here, tfm-rubygem-concurrent-ruby?